packages/components/CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+-	`GradientPicker`: Add disableAlpha prop ([#66974](https://github.com/WordPress/gutenberg/pull/66974))
+-	`CustomGradientPicker`: Add disableAlpha prop ([#66974](https://github.com/WordPress/gutenberg/pull/66974))
```

Comment:
I think we can tweak the spacing a bit to be consistent with the rest of the CHANGELOG:

```suggestion
-   `GradientPicker`: Add disableAlpha prop ([#66974](https://github.com/WordPress/gutenberg/pull/66974))
-   `CustomGradientPicker`: Add disableAlpha prop ([#66974](https://github.com/WordPress/gutenberg/pull/66974))
```